src/microsAuthentication/content.md

```diff
@@ -385,16 +384,48 @@ To issue a request to the Token Review API, you need to create a TokenReview res
 ```yaml|title=token.yaml
 kind: TokenReview
 apiVersion: authentication.k8s.io/v1
+metadata:
+  name: test
 spec:
-  token: <the token that you want to validate>
+  token: <replace with the token to validate>
 ```
 
 AMIT: this should be refactored. We can use Kubectl + impersonation.
 
 > I am not sure how we can make an API call using impersonation. Quick google didn't return anything
-> Also I would prefer this example to be as it is as it shows how this would work in an application running in a 
+> Also I would prefer this example to be as it is as it shows how this would work in an application running in a
 > pod and flows nicely into the service implementation in Go.
 
+AMIT: This is where we shine :)
+It's a POST request... so it should be `kubectl apply`.
+
+```terminal|command=1|title=bash
+kubectl apply -v=8 -f token.yaml
+tokenreview.authentication.k8s.io/test created
+kubectl get tokenreview
+Error from server (MethodNotAllowed): the server does not allow this method on the requested resource
+kubectl get tokenreview.authentication.k8s.io
+Error from server (MethodNotAllowed): the server does not allow this method on the requested resource
+kubectl describe tokenreview.authentication.k8s.io
+Error from server (MethodNotAllowed): the server does not allow this method on the requested resource
+```
+
+I was ready to give up when I tried adding `-v8` to the apply request.
+
+The response contains the JSON payload.
+
+Next, `kubectl apply -h` reveals that there's a `-o` flag for the output.
+
+And... voila! Validate a token with only kubectl:
+
+```terminal|command=1|title=bash
+kubectl apply -o yaml -f token.yaml
+```
+
+You're issueing a request using your own account (cluster admin) and validating the token for a Service account.
+
+It's easier to explain, and we skip the part where we connect to the API. The same part is skipped in the Go client too (it's part of the SDK).
```

Comment:
Nice find! I think i see your point here - not showing you to access the API directly. 

How about we use kubectl to show the token review operation but use the token for the service token instead via the --token argument?

In addition can we still keep the curl bit?

Reply:
Could you show me an example?

Reply:
> How about we use kubectl to show the token review operation but use the token for the service token instead via the --token argument?

# Get the token from the secret object
kubectl apply -f t.yaml --token <token from above>